src/AssemblyaiApi/Transcripts/requests/WordSearchParams.cs

```diff
@@ -0,0 +1,9 @@
+namespace AssemblyaiApi;
+
+public class WordSearchParams
+{
+    /// <summary>
+    /// Keywords to search for
+    /// </summary>
+    public string? Words { get; init; }
```

Comment:
Non-blocking: Type should be `IEnumerable<string>`, this is also wrong in other SDKs

Reply:
@Swimburger this is likely an issue in the OAI spec in that case

Reply:
We have the following request parameter in the spec:
```yaml
        - name: words
          x-label: Words
          in: query
          description: Keywords to search for
          required: true
          style: form
          explode: false
          schema:
            type: array
            items:
              x-label: Word
              type: string
```

We're using `style: form` with `explode: false` which should result in a comma separated querystring parameter when serialized.
I don't see an issue with the spec.

<img width="1077" alt="Screenshot 2024-05-28 at 9 40 20 AM" src="https://github.com/AssemblyAI/assemblyai-csharp-sdk/assets/3382717/059ab3f4-f910-4763-a9bd-acca46f8e15c">